params/protocol_params.go

```diff
@@ -37,7 +37,6 @@ const (
 
 	Sha3Gas          uint64 = 30 // Once per SHA3 operation.
 	Sha3WordGas      uint64 = 6  // Once per word of the SHA3 operation's data.
-	Keccak256Gas     uint64 = 30 // Once per KECCAK256 operation.
```

Comment:
why remove?

Reply:
`Sha3Gas` was renamed to `Keccak256Gas` in geth but we have not adopted this change yet.